ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -7,8 +7,8 @@
 
 #### L2
 
-| Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | Channel-Group |
-| --------- | ----------- | ---- | ----- | ----------- | ----------- | ------------- |
+| Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | Channel-Group | Load Interval |
```

Comment:
I don't think we need to document this small config knob. We try to only document the important interface-level knobs, since the table would explode if we had all the settings in their own column.
That said, when changing general documentation templates, you have to update _all_ molecule scenarios, since it will affect all devices. This also means it will affect every deployment out there.

Reply:
removed back to previous version